### PR TITLE
[EAM-3122] All custom tab grids displayed in the Position page

### PR DESCRIPTION
**Pull request by purquijo**
**Limitations:**

- The tab grids displayed are limited to those following the pattern: parentScreenCode_X* (e.g., OSOBJ_XAC). Therefore, custom tab grids must adhere to this naming convention to be displayed. EAM enforces this naming convention. Other screens should not adopt this structure.
- Links are displayed only for fields with the aliases: caseno, equipmentno, workorderno, partno.
- Custom tab grids are exclusively visible on the Position page. However, they can be added to other equipment pages, which should be easy.
- The param must be one of these: "position", "obj_code", "main_eqp_code", "OBJ_CODE", "object". This can be expanded. Determined by eam-components.

### CODE REVIEW

**Comment by purquijo** [comment]
The component has been added to: 
- Asset
- Location
- Position
- System
- Part
- Workorder

**Comment by ricardo-j-almeida** [comment]
This MR is marked as WIP, is it done?

**Comment by purquijo** [comment]
> Depends on [PR](https://github.com/cern-eam/eam-wshub-core/pull/95) and [PR](https://github.com/cern-eam/eam-components/pull/163)

@ricardo-j-almeida it was because of this

File: src/ui/pages/equipment/location/Location.js [hunk 1]
```diff
@@ -77,6 +81,36 @@ export default Location = (props) => {
...
+                    </EAMGridTab>
+                ,
+                column: 2,
+                order: 30 + 5 * index,
+                summaryIcon: GridOnIcon,
+                ignore: !tab.tabAvailable,             
+                initialVisibility: tab.alwaysDisplayed    
+            });
+        })
+    }
```
Comment (ricardo-j-almeida) [inline src/ui/pages/equipment/location/Location.js:112]: This function is really similar throughout. Can we externalize it?

Comment (purquijo) [reply src/ui/pages/equipment/location/Location.js]: @ricardo-j-almeida done

**Pull request merged**